aten/src/ATen/core/interned_strings_class.h

```diff
@@ -0,0 +1,39 @@
+#include <cstdint>
+#include <iostream>
+#include <mutex>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <vector>
+#include "ATen/core/Error.h"
+#include "ATen/core/interned_strings.h"
+#include "ATen/core/optional.h"
+#include "string.h"
+
+namespace torch {
+namespace jit {
+
+struct CAFFE2_API InternedStrings {
```

Comment:
Sorry, I am going to be a documentation policeman for a while but this class has zero comments :)

Reply:
that's true https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/interned_strings_class.h

Reply:
```
// This class represents the global state that keeps track of interned strings.
// It maintains mappings of strings to their interned representation (Symbol)
// and vice versa.
```